src/ScriptExport/Options.hs

```diff
@@ -138,19 +147,23 @@ serverOpt =
           <> Opt.value 3600
           <> Opt.help "How many seconds should the server keep cached results available for."
       )
-
-opt :: Opt.Parser Options
-opt =
-  FileOption <$> fileOpt
-    <|> ServerOption <$> serverOpt
+    <**> Opt.helper
 
 {- | Parse 'Options' from the command line arguments.
 
-     @since 2.0.0
+     @since 2.2.0
 -}
 parseOptions :: IO Options
 parseOptions = Opt.execParser p
   where
+    opt =
+      Opt.subparser . foldMap mkSubcommand $
+        [ (FileOption <$> fileOpt, "file", "Complie scripts using file IO")
+        , (ServerOption <$> serverOpt, "server", "Start script server")
```

Comment:
How about calling it `serve` instead of `server`?

Reply:
Yeah, good idea